src/ExplainerAtom.tsx

```diff
@@ -48,16 +51,22 @@ const Title = ({ title }: { title: string }) => (
     </h3>
 );
 
-const Body = ({ html }: { html: string }) => (
-    <div
-        css={css`
-            ${textSans.small({ fontWeight: 'light', lineHeight: 'tight' })}
-        `}
-        dangerouslySetInnerHTML={{
-            __html: html,
-        }}
-    />
-);
+const Body = ({ html }: { html: string }) => {
+    console.log(html);
```

Comment:
```suggestion
```

Reply:
Actually, you probably want to revert this back to the `=> ()` rather than `return`?

Reply:
Yeah silly mistake, just fixed